e2e/testsuite/testsuite.go

```diff
@@ -240,8 +292,9 @@ func (s *E2ETestSuite) GetPathName(idx int64) string {
 }
 
 // generatePath generates the path name using the test suites name
-func (s *E2ETestSuite) generatePath(ctx context.Context, ibcrelayer ibc.Relayer) string {
-	chainA, chainB := s.GetChains()
+func (s *E2ETestSuite) generatePath(ctx context.Context, ibcrelayer ibc.Relayer, chainAIdx, chainBIdx int) string {
```

Comment:
Could we update godoc? The idea here is that if we want to generate a path between chainA and chainC, we provide the args 0, 2?